### DISCUSSION TITLE
Support for K3s on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by BGOtura** [hubblo-org/scaphandre#281] [Q&A]
Hi!

Is there any support or planned support for power monitoring of a K3s cluster running on a raspberry pi? (ARM architecture)

Thanks!
Borja

**Comment by BGOtura** [comment]
I see there is a thread about ARM here:  #35 , but not specifically about K3s

**Comment by TheElectronWill** [comment]
Hi!
Have you tried to deploy Scaphandre's Helm chart on K3S? If I understood correctly, [K3S supports Helm out of the box](https://docs.k3s.io/helm), so maybe it already works. (getting measurements on ARM is another story of course)

**Reply by BGOtura** [reply]
Hi Guillaume,

I haven´t yet. So far, all K3s we have are running on Raspberry Pis, so they dont have the Intel_RAPL sensor.

Borja
